### ISSUE TITLE
Cool Website: Setup Basic Features

### ISSUE THREAD

**Issue by cdcline** [cdcline/demo-website#115]
I'd like to make a cool website! But I'm starting from nothing. First we'll need to setup a basic framework to display some content in HTML, CSS and to mess with it using JS.

## Plan A
- [x] #1 
- [x] #26
- [x] #32 
- [x] #49 
- [x] #22

## Bonus Points
These would be nice to have but really aren't required for the basic idea. More features => more complex code => harder to understand and that's kinda counter to what this project is about. These are in a rough order of priority once Plan A is done.
- [x] #14
- [x] #51 
- [x] #36
- [x] #39 
- [ ] #35
- [ ] #15
- [ ] #45 
- [x] #43 
- [ ] #44
- [ ] #17
- [x] #13 
- [x] #16
- [ ] #34
- [ ] #33
- [x] #11 
- [ ] #40 

## Estimate
2 weeks to get plan A deployed

## Future Projects
 * Do it all again in react
 * Do it in not PHP
   * Go (?)
   * Python

**Comment by cdcline** [comment]
This was actually moved from [another issue](https://github.com/cdcline/demo-website/issues/12) that morphed into a general overview of the project instead of just this initial part of it.
